_data/nav.yml

```diff
@@ -33,7 +35,14 @@
     - subtitle: "CTF"
       subhref: "/ctf"
 
-- title: "Work With Us"
+- title: "Program"
+  href: "/cfp"
+  subcategories:
+    - subtitle: "Call For Participation"
+      subhref: "/cfp"
+
+- title: "Join Us"
+  href: "/volunteers"
```

Comment:
This may cause some confusion and cause people not to see `/jobs`. Assuming we're ok with that?

Reply:
At least for now, yes.  I'd rather have people sign up to be a volunteer and ask about how to become staff, than have a bunch of folks clutter up the staff applications when they only want to volunteer.